### PR TITLE
Updating debugger.md to be a little clearer

### PR DESCRIPTION
**Pull request by joseph4tw**
Hello,

I got confused with this line:
> You can start from scratch by creating an empty project with dotnet new:

I wasn't sure if I should use the VS Code command palette `Ctrl+p` for `dotnet new` when I first read it, so I added some more text to say to use the VS Code Terminal or Command Prompt for the following steps.

I also changed some of the `quoted` text to be `preformatted` instead because it seemed to be consistent to have the user `click` or `type` things that were in `preformatted text`.

I hope this helps other readers.

### CODE REVIEW

File: debugger.md [hunk 1]
```diff
@@ -27,9 +27,9 @@ Install the .NET Core command line tools (CLI) by following the installation par
 **OSX:** .NET Core requires openSSL to work. Don't forget this! Execute: `brew install openssl`
 
 ##### 3: Install C# Extension for VS Code
-Open the command palette in VS Code (F1) and type "ext install C#" to trigger the installation of the extension. VS Code will show a message that the extension has been installed and it will restart.
+Open the command palette in VS Code (`F1`) and type `ext install C#` to trigger the installation of the extension. VS Code will show a message that the extension has been installed and it will restart.
```
Comment (DustinCampbell) [inline debugger.md]: I would use `<kbd>F1</kbd>`.

File: debugger.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ Install the .NET Core command line tools (CLI) by following the installation par
...
 
-If you have previously installed the C# extension, make sure that you have a recent version. You can check this by opening the command palette (F1) and running 'Extensions: Show Installed Extensions'.
+If you have previously installed the C# extension, make sure that you have a recent version. You can check this by opening the command palette (`F1`) and running `Extensions: Show Installed Extensions`.
```
Comment (DustinCampbell) [inline debugger.md]: Ditto.

File: debugger.md [hunk 3]
```diff
@@ -62,30 +62,30 @@ If you open the folder containing your project.json, the C# extension can automa
...
+**.vscode/tasks.json**: Start with [this example](https://raw.githubusercontent.com/wiki/OmniSharp/omnisharp-vscode/ExampleCode/tasks.json) which configures VS Code to launch `dotnet build`. Update the `cwd` property if your project isn't in the root of the open folder. If you don't want to build from VS Code at all, you can skip this file. If you do this, you will need to comment out the `preLaunchTask` from .vscode/launch.json when you create it.
 
-**.vscode/launch.json**: When you want to start debugging, press the debugger play button (or hit F5) as you would normally do. VS Code will provide a list of templates to select from. Pick ".NET Core" from this list and the edit the 'program' property to indicate the path to the application dll or .NET Core host executable to launch. For example:
+**.vscode/launch.json**: When you want to start debugging, press the debugger play button (or hit `F5`) as you would normally do. VS Code will provide a list of templates to select from. Pick ".NET Core" from this list and the edit the `program` property to indicate the path to the application dll or .NET Core host executable to launch. For example:
```
Comment (DustinCampbell) [inline debugger.md]: Ditto. `<kbd>F5</kbd>` to get <kbd>F5</kbd>. Also, "press" instead of "hit". :smile:

File: debugger.md [hunk 4]
```diff
@@ -62,30 +62,30 @@ If you open the folder containing your project.json, the C# extension can automa
...
 
 	"configurations": [
 		{
 			...
 			"program": "${workspaceRoot}/MyLaunchingProject/bin/Debug/netcoreapp1.0/MyLaunchingProject.dll",
 
 ##### 4: Start debugging
-Your project is now all set. Set a breakpoint or two where you want to stop, click the debugger play button (or hit F5) and you are off.
+Your project is now all set. Set a breakpoint or two where you want to stop, click the debugger play button (or hit `F5`) and you are off.
```
Comment (DustinCampbell) [inline debugger.md]: Same thing here.

File: debugger.md [hunk 5]
```diff
@@ -132,7 +132,7 @@ The C# debugger supports attaching to processes. To do this, switch to the Debug
 
 ![Debug launch configuration drop down](https://raw.githubusercontent.com/wiki/OmniSharp/omnisharp-vscode/images/debug-launch-configurations.png)
 
-Select the '.NET Core Attach' configuration. Clicking the play button (or pressing F5) will then try to attach. In launch.json, if `processId` is set to `"${command.pickProcess}"` this will provide UI to select which process to attach to.
+Select the '.NET Core Attach' configuration. Clicking the play button (or pressing `F5`) will then try to attach. In launch.json, if `processId` is set to `"${command.pickProcess}"` this will provide UI to select which process to attach to.
```
Comment (DustinCampbell) [inline debugger.md]: And here.

Comment (joseph4tw) [inline debugger.md]: 👍 works for me! I'll make the edits now.

**Comment by joseph4tw** [comment]
@DustinCampbell PR updated. Thanks for reviewing so quickly!

**Comment by DustinCampbell** [comment]
I only gave it a quick look. The debugger guys will want to review. cc @gregg-miskelly, @rajkumar42.

File: debugger.md [hunk 6]
```diff
@@ -39,21 +39,21 @@ The first time that C# code is opened in VS Code, the extension will download th
 The following steps have to executed for every project. 
 
 ##### 1: Get a project
-You can start from scratch by creating an empty project with `dotnet new`:
+You can start from scratch by creating an empty project with `dotnet new`. Begin by opening the terminal in Visual Studio Code (`View->Integrated Terminal`) or use Command Prompt and type these commands:
 
     cd ~
```
Comment (rajkumar42) [inline debugger.md]: "cd ~" won't work on cmd.

**Comment by gregg-miskelly** [comment]
LGTM. Thanks for doing this!

File: debugger.md (hunk 6 above)
Comment (joseph4tw) [inline debugger.md]: Good point. I'll make one more revision and remove the `or use Command Prompt` part.

**Pull request merged**